assets/tests/components/notificationModals/inactiveStateModal.test.tsx

```diff
@@ -69,18 +69,18 @@ describe("InactiveNotificationModal", () => {
 
   test("renders loading message", () => {
     ;(useMinischeduleRuns as jest.Mock).mockImplementationOnce(() => undefined)
-    const tree = renderer
-      .create(<InactiveNotificationModal notification={notification} />)
-      .toJSON()
-    expect(tree).toMatchSnapshot()
+    const result = render(
+      <InactiveNotificationModal notification={notification} />
+    )
+    expect(result.baseElement).toMatchSnapshot()
   })
 
   test("renders for a notification with no runs", () => {
     ;(useMinischeduleRuns as jest.Mock).mockImplementationOnce(() => [])
-    const tree = renderer
-      .create(<InactiveNotificationModal notification={notification} />)
-      .toJSON()
-    expect(tree).toMatchSnapshot()
+    const result = render(
+      <InactiveNotificationModal notification={notification} />
+    )
+    expect(result.baseElement).toMatchSnapshot()
```

Comment:
suggestion(non-blocking): I don't think it's worth blocking over, but [Kent C Dodd's "Common mistakes with React Testing Library"](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-wrapper-as-the-variable-name-for-the-return-value-from-render) recommends

> Advice: destructure what you need from render or call it view.

I usually prefer to destructure the needed items

```suggestion
    const { baseElement } = render(
      <InactiveNotificationModal notification={notification} />
    )
    expect(baseElement).toMatchSnapshot()
```